content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Change Notes
+description: Istio 1.7 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** config option `values.global.proxy.holdApplicationUntilProxyStarts`,
+which causes the sidecar injector to inject the sidecar at the start of the
+pod's container list and configures it to block the start of all other
+containers until the proxy is ready.  This option is disabled by default.
+ ([Issue #11130](https://github.com/istio/istio/issues/11130))
+- **Added** SDS support for Client Certificate and CA certificate used for [TLS/mTLS Origination from Egress Gateway](/docs/tasks/traffic-management/egress/egress-gateway-tls-origination-sds/) using `DestinationRule`.
+  ([Issue #14039](https://github.com/istio/istio/issues/14039))
+
+## Security
```

Comment:
@JimmyCYJ @nschhina Can you add a bullet for SDS support on Egress gateway with the issue number?

Reply:
Oh I see it listed in traffic management.